sass/_Quickview.scss

```diff
@@ -178,7 +178,7 @@
 
 .coveo-quickview-time {
   font-size: 12px;
-  color: #b4b4b4;
+  color: $color-dark-grey;
```

Comment:
Is there any other recommendation on which color should be used? Should I contact UX?

Reply:
👍

Reply:
Good point about getting a closer color, I tried `$color-light-grey` and it seems to be too light. `$color-grey` still meets the minimum required contrast, so I'll use that instead.